apache/trafficcontrol
# Rewrite get to_extensions

## What does this PR (Pull Request) do?


- [x] This PR fixes #3843 GET portion

This PR rewrite ` GET to_extensions` to GO and as apart of the rewrite it now includes in the response configured TO_Plugins. Previously it would dynamically load in perl extensions at run time which is not wanted as we move off Perl.

## Which Traffic Control components are affected by this PR?


- Documentation
- Traffic Ops


## What is the best way to verify this PR?

* Make a request to `GET /to_extension` make sure it works as expected / run the API tests.
* Configure a TO Plugin and make a request to `GET /to_extension` and it should include information about that Plugin

## If this is a bug fix, what versions of Traffic Control are affected?



## The following criteria are ALL met by this PR




## Additional Information

Dependent on #4327
